pkg/scheduler/internal/cache/interface.go

```diff
@@ -59,6 +59,10 @@ import (
 // - Both "Expired" and "Deleted" are valid end states. In case of some problems, e.g. network issue,
 //   a pod might have changed its state (e.g. added and deleted) without delivering notification to the cache.
 type Cache interface {
+	algorithm.PodLister
```

Comment:
Why are we use the interface instead of method signatures here? We could revert this change to solve @bsalamat's comment [here](https://github.com/kubernetes/kubernetes/pull/80220/files#r305156226)

Reply:
A signature should be written once. We need to give more thought to this.